addons/calendar/models/calendar_event.py

```diff
@@ -406,13 +406,14 @@ def _compute_recurrence(self):
         defaults = self.env['calendar.recurrence'].default_get(recurrence_fields)
         for event in self:
             if event.recurrency:
+                event.update(defaults)
```

Comment:
I would add a small comment to explain that the default recurrence values are needed to retrieve correct event_values. Otherwise this is not super clear.